content/en/docs/collector/configuration.md

```diff
@@ -67,8 +67,7 @@ service:
       exporters: [otlp]
 ```
 
-Note that the same receiver, processor, exporter and/or pipeline can be defined
-more than once. For example:
+Note that receivers, processors, exporters and/or pipelines are defined via component identifiers in `type[/name]` format (e.g. `otlp` or `otlp/2`).  Components of a given type can be defined more than once as long as the identifiers are unique. For example:
```

Comment:
Double space :see_no_evil:

`).  Components of a given`

Reply:
Good catch! I actually dislike the (legacy) double space style, so I would consider this a typo. 😅